docs/source/releases/v1_11_3a0.rst

```diff
@@ -138,3 +138,23 @@ Include script that sets explicit paths for geoips environment setup.
 ::
 
   new: setup/USER_MODIFY_config
+
+Refactoring Updates
+===================
+
+Change the specification of tpw_cimss and tpw_purple colormappers
```

Comment:
@evrose54  I usually try to also add a line of summary at the top of the release notes, in addition to adding the details under the "Refactoring Updates" header.  Just makes it easier to quickly see at the top what all changed in the current version.

Often this is just a copy/paste of the title, sometimes a shorter one line summary of multiple changes.

Feel to resolve if that makes sense to you.

Reply:
Makes sense to me. Thanks for making that change as well. I'll do that next time!